mslib/msui/mscolab.py

```diff
@@ -693,7 +697,35 @@ def after_login(self, emailid, url, r):
 
             self.signal_login_mscolab.emit(self.mscolab_server_url, self.token)
 
-    def fetch_gravatar(self, refresh=False):
+    def set_profile_pixmap(self, img_data):
+        pixmap = QPixmap()
+        pixmap.loadFromData(img_data)
+        resized_pixmap = pixmap.scaled(64, 64)
+
+        if (hasattr(self, 'profile_dialog') and self.profile_dialog is not None and
```

Comment:
here you can add also a Todo
e.g. verify by a test if the condition can be simplified, by if self.profile_dialog is not None

Reply:
well, I tried it and it did not work

Reply:
that's not for now, this needs a test coverage